Rakefile

```diff
@@ -1,8 +1,12 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require 'bundler/gem_tasks'
+require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
```

Comment:
Reasoning for not using single quotes?  Rubocop does not have a rule for forcing double quotes on non-interpolationed strings